lib/index.js

```diff
@@ -5,6 +5,7 @@ var Matcher = require('minimatch').Minimatch;
 var unique = require('uniq');
 var read = require('fs').readFileSync;
 var loadMetadata = require('read-metadata').sync;
+var path = require('path');
```

Comment:
This is superfluous, looks like it was added and left behind when not needed anymore.